spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -154,6 +155,21 @@ def submit_prefilled_otp_code
         visit profile_path
         expect(current_path).to eq root_path
       end
+
+      it 'leaves the user on the lockout page during the entire lockout period', js: true do
+        Timecop.freeze do
+          allow(Figaro.env).to receive(:session_check_frequency).and_return('0')
+          allow(Figaro.env).to receive(:session_check_delay).and_return('0')
+          five_minute_countdown_regex = /4:5\d/
+          allow_any_instance_of(User).to receive(:max_login_attempts?).and_return(true)
```

Comment:
yes we could, although since these are feature specs it's nice to have at least one test that goes through the entire flow manually, IMHO